File: DSA/Assignment_3/rotate_90.c
#include <stdio.h>
int main()
{
    int m,n;
    printf("Enter row and column size:");
    scanf("%d %d",&m,&n);
    int arr[m][n],rot[m][n];
    for(int i=0;i<m;i++)
    {
        for(int j=0;j<n;j++)
        {
            printf("Enter element:");
            scanf("%d",&arr[i][j]);
        }
    }

    for(int i=0;i<m;i++)
    {
        for(int j=0;j<n;j++)
        {
            rot[j][n-i-1]=arr[i][j];
        }
    }

    printf("Rotated matrix:\n");
    for(int i=0;i<m;i++)
    {
        for(int j=0;j<n;j++)
        {
            printf("%d ",rot[i][j]);
        }
        printf("\n");
    }

    return 0;
}

File: DSA/Assignment_3/rotate_90.py
m=int(input("Enter row size:"))
n=int(input("Enter column size:"))
arr=[]
rot=[]
for i in range(m):
    a=[]
    for j in range(n):
        a.append(int(input("Enter element:")))
    arr.append(a)

rot=[[0 for _ in range(n)]for _ in range(m)]
for i in range(m):
    for j in range(n):
        rot[j][n-i-1]=arr[i][j]

print("Rotated Matrix:")
for i in rot:
    